nodebuilder/state/cmds/state.go

```diff
@@ -30,24 +35,27 @@ func init() {
 	)
 }
 
-var stateCmd = &cobra.Command{
+var Cmd = &cobra.Command{
 	Use:   "state [command]",
 	Short: "Allows interaction with the State Module via JSON-RPC",
 	Args:  cobra.NoArgs,
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
i'm not sure how scoping would work, but wondering if would we could this PersistenPreRunE func somewhere else and return it from a util package somewhere?

Reply:
I thought about it, but it will end up with one more constructor that creates rpc client. I'm not sure that it is worth it